Components/IndividualList/IndividualList.js

```diff
@@ -3,7 +3,7 @@ import { View, Text, StyleSheet, TextInput } from "react-native";
 import { TouchableHighlight } from "react-native-gesture-handler";
 import { connect } from "react-redux";
 import { loadTasks } from "../../actions";
-import { fetchTasks, postTask, patchTask, deleteTask } from "../../apiCalls";
+import { fetchTasks, postTask, patchTask, deleteTask } from "../../Utils/apiCalls";
```

Comment:
@kalex19 ,

So far I've added all my fetches in a file called apiCalls.js . I see that you've added yours in a Utils folder. Should we put them all in the same place or, should we leave them separate, as they are?

Reply:
I think ideally they should be in the same place. We can refactor later.